riotfile.py

```diff
@@ -2600,6 +2600,16 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
                 ),
             ],
         ),
+        Venv(
+            name="datadog_lambda",
+            command="pytest {cmdargs} tests/contrib/datadog_lambda",
+            pys=select_pys(min_version="3.7", max_version="3.9"),
```

Comment:
Does Lambda only support these Python versions? If so, quite interesting how far behind they are. If not, can we add all the supported versions.

Reply:
wow lol